app/routes/wallet/jws.py

```diff
@@ -23,15 +23,36 @@
     "/sign",
     response_model=JWSCreateResponse,
     summary="Sign JWS",
-    description="""
-Sign JSON Web Signature (JWS)
-
-See https://www.rfc-editor.org/rfc/rfc7515.html for the JWS spec.""",
 )
 async def sign_jws(
     body: JWSCreateRequest,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> JWSCreateResponse:
+    """
+    Sign JSON Web Signature (JWS)
+    ---
+
+    The difference between the did and verification_method fields is
+    that if the `did` field is used, acapy will make an educated guess
+    about which key associated with the did to use to sign the jwt.
+
+    While with the `verification_method` field, the user is explicitly
+    specifying which key to use to sign the jwt.
+
+    See https://www.rfc-editor.org/rfc/rfc7515.html for the JWS spec.
+
+    Request body:
+    ---
+        JWSCreateRequest: The JWS to sign.
+            did: str: The DID to sign the JWS with.
+            header: dict: The header of the JWS.
+            payload: dict: The payload of the JWS.
+            verification_method: str: The verification (did with key to use) method to use.
+
+    Returns:
+    ---
+        JWSCreateResponse: The signed JWS.
```

Comment:
The format used in other routes looks a bit better imo (instead of writing description right next to attribute):
```
    Returns:
    ---
        JWSCreateResponse
            A JWS string representing the signed JSON Web Signature. This string is the compact serialization
                   format of the JWS, which includes the encoded header, payload, and signature. 
```
(I asked GPT to generate that 😄 just some inspiration, we don't have to use that word for word)